.github/workflows/aws_openshift_rosa_hcp_single_region_golden.yml

```diff
@@ -96,3 +96,22 @@ jobs:
                       ```sh
                       just regenerate-golden-file "${{ env.MODULE_DIR }}" "${{ env.S3_BUCKET_REGION }}" "${{ env.S3_BACKEND_BUCKET }}" "${{ env.S3_BUCKET_KEY }}"
                       ```
+                  comment-body-no-diff: ✅ No diff, you can ignore this comment and below footer.
+
+            - name: Delete specific comment containing the text and post diff
+              if: always() && github.event_name == 'pull_request'
```

Comment:
I didn't check all your changes but you could have it easier by using the output of the action

https://github.com/int128/diff-action

The above action would need a name but then you could just call it on `steps....outputs.different"

Reply:
talked about it, all good